private/buf/buffetch/internal/reader.go

```diff
@@ -348,6 +356,89 @@ func (r *reader) getDirBucket(
 	)
 }
 
+func (r *reader) getSingleFileBucket(
+	ctx context.Context,
+	container app.EnvStdinContainer,
+	singleFileRef SingleFileRef,
+	terminateFileNames []string,
+) (ReadBucketCloser, error) {
+	if !r.localEnabled {
+		return nil, NewReadLocalDisabledError()
+	}
+	terminateFileNames = append(terminateFileNames, bufconfig.ExternalConfigV1FilePath)
```

Comment:
We need to talk about this - `buffetch/internal` was meant to be independent of any buf-specific concepts such as the config file name, this was why we added `terminateFileNames`

Reply:
Yeah -- as per the suggestion below, we ended up going with a priority order for terminate files, just added that :)